README.md

```diff
@@ -5,11 +5,12 @@
 [![Build Status](https://circleci.com/gh/windmilleng/tilt/tree/master.svg?style=shield)](https://circleci.com/gh/windmilleng/tilt)
 [![GoDoc](https://godoc.org/github.com/windmilleng/tilt?status.svg)](https://godoc.org/github.com/windmilleng/tilt)
 
-Local Kubernetes development with no stress.
+Kubernetes for Prod, Tilt for Dev
 
-[Tilt](https://tilt.dev) helps you develop your microservices locally.
-Run `tilt up` to start working on your services in a complete dev environment
-configured for your team.
+Modern apps are made of multiple services: a database, an API server, a web bundler.
```

Comment:
Can we make this more generic? `a frontend, a backend, a database`?

Reply:
ya, i struggled with this a bit.

I disagree that "a frontend, a backend, a database" evokes a neutral, generic architecture. It evokes a very particular architecture. So I wanted that seemed like it applied even if you were doing something like Jamstack or Celery tasks. wdyt?

Reply:
I think it doesn't work.

`Modern apps are made of a myriad of different services in constant communication.` ?

Reply:
ooh, i like the idea of leaning into the idea of escalating complexity.

How about:

```
Modern apps are made of too many services. They're everywhere and in constant communication. 

[Tilt](https://tilt.dev) powers multi-service development so you can keep an eye on them.
```

too much?

Reply:
`so you can keep an eye on them` feels like passive voice to me.

I'd go for something like `and keeps them in line.`, `and makes sure they behave!` Something like that?

The gist of the edit is that I don't want "to be able to do something if I so please." A more active sentence construction works best for copy editing. (Think movie-trailer voice.)

Reply:
haha, i :green_heart: both of those suggestions, i went with the more sassy one!